hack/jenkins/update-jobs.sh

```diff
@@ -42,4 +42,4 @@ fi
 
 docker exec job-builder git checkout master
 docker exec job-builder git pull
-docker exec job-builder jenkins-jobs test ${config_dir}
+docker exec job-builder jenkins-jobs update ${config_dir}
```

Comment:
It's not built into JJB, but it's not too difficult to do this manually. I'm not sure if it's necessary, though. While we're adding jobs for the first time we need to make sure to backup the old configs and pay attention to the diffs. Once they're all in, however, if something goes haywire we can just revert to the last working version. Warning about a big diff doesn't matter much, then.